From 6d25b42f1f03bec5791000590bd24f9e2ab4d431 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 13:00:33 +0800
Subject: [PATCH 1/1] Test case for driver .vfr file

Description: Change the default ID of 'MyStandardDefault' value in the MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr b/MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr
index 65a65d4d1dc4..7e6da0341fc4 100644
--- a/MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr
+++ b/MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr
@@ -98,7 +98,7 @@ formset
 
   defaultstore MyStandardDefault,
     prompt      = STRING_TOKEN(STR_STANDARD_DEFAULT_PROMPT),
-    attribute   = 0x0000;                         // Default ID: 0000 standard default
+    attribute   = 0x0002;                         // Default ID: 0000 standard default
 
   defaultstore MyManufactureDefault,
     prompt      = STRING_TOKEN(STR_MANUFACTURE_DEFAULT_PROMPT),
-- 
2.27.0.windows.1

